### ISSUE TITLE
Toolbar, UI controls too small on 4k monitor.

### ISSUE THREAD

**Issue by 100amps** [fontforge/fontforge#4649]
Using 2020/11/07 Windows.exe
First time installing ff. The GUI controls don't scale to fit the large-ish (28") high res.(4k) screen size. The toolbar was unusably tiny. Each tool was about 3mm square (in physical screen space), too small to even make out what the icons are. 

I can see microscopic x,y text and other parameters along the top of the glyph edit screen but again, too tiny to read. 

I tried playing with the x configuration but couldnt find any settings that helped.

**Comment by jtanx** [comment]
Try set `GDK_SCALE`.

e.g. open the interactive console and do

```
set GDK_SCALE=2
fontforge
```

You can modify fontforge.bat to make it permanent

**Comment by 100amps** [comment]
That helped a lot, thanks.

**Comment by 100amps** [comment]
I really should not have closed this issue. The GDK_SCALE=2 setting made most of the UI controls a useable size, which did solve my immediate problem, but there are still serious scale problems it didn't resolve. 
  - the mouse pointer is still very tiny.
  - the mouse pointer clicking target area is also very tiny, making it very hard to select a bezier handle. It's frustrating to miss the spot 4-5 times before hitting it -- for every single adjustment.
  - the glyph rendering is scaled relatively to a lower res. so that the antialiasing became very obvious, and generally quite blurry.

In general, a more effective solution is needed to counter-scale the UI as screen resolution increases. It should become sharper without the controls shrinking to the same degree. This problem will only get worse as screen resolution increases.

**Comment by 100amps** [comment]
Screenshot attached to illustrate the scaling problem. Scale the image to fully fit into your screen. Chrome/Gmail UI included at same resolution for comparison of control sizes. The Google stuff is scaled fine and comfortable.

<img width="1910" alt="FF-window-scale" src="https://user-images.githubusercontent.com/18044808/111846831-d8f78600-88c4-11eb-90ed-5391048fc2fa.png">

**Comment by skef** [comment]
I don't have any of these problems with GDK_SCALE=2 at 4k on Arch Linux. Therefore they are probably Windows-specific issues with GDK, or would need to be fixed with windows-specific handling. I'm afraid that's unlikely to happen. Maybe a future update of GDK will help resolve things.

**Comment by BusterDev** [comment]
Bit of a late reply, but maybe useful for someone...
This problem can be solved by changing the application DPI settings in Windows.
Navigate to the Fontforge executable (default is C:\Program Files (x86)\FontForgeBuilds\run_fontforge.exe), then right-click.
Select "Properties", then the "Compatibility" tab, then click on the "Change DPI settings" button.
Check the "Override high DPI scaling behavior". Select "System (Enhanced)" in the "Scaling performed by" dropdown menu.
You're done!

**Comment by visually-creative** [comment]
> Bit of a late reply, but maybe useful for someone... This problem can be solved by changing the application DPI settings in Windows. Navigate to the Fontforge executable (default is C:\Program Files (x86)\FontForgeBuilds\run_fontforge.exe), then right-click. Select "Properties", then the "Compatibility" tab, then click on the "Change DPI settings" button. Check the "Override high DPI scaling behavior". Select "System (Enhanced)" in the "Scaling performed by" dropdown menu. You're done!

Yes! Thank you so much. Works great, finally I can see what I'm doing better.
